airflow/models/abstractoperator.py

```diff
@@ -154,30 +154,31 @@ def get_direct_relative_ids(self, upstream: bool = False) -> set[str]:
             return self.upstream_task_ids
         return self.downstream_task_ids
 
-    def get_flat_relative_ids(
-        self,
-        upstream: bool = False,
-        found_descendants: set[str] | None = None,
-    ) -> set[str]:
-        """Get a flat set of relative IDs, upstream or downstream."""
+    def get_flat_relative_ids(self, upstream: bool = False) -> set[str]:
```

Comment:
Should we kwarg only it while we are at it? There is only 1 use that would need adjusting.

Reply:
done